mslib/msui/__init__.py

```diff
@@ -54,7 +54,7 @@ class MissionSupportSystemDefaultConfig(object):
     filepicker_default = "default"
 
     # dir where mss output files are stored
-    data_dir = "~/mssdata"
+    data_dir = os.path.join(os.path.expanduser("~/"), "Documents", "mss")
```

Comment:
this should be an fs path.

and we don't have "Documents" on all systems. For example this depends if you do a desktop or server installation. I don't expect it in a docker installation.

Reply:
I do not understand how to properly use fs. See current issue.

This particular path should point to a sensible location to store ftml files, which would be the Documents folder on Windows and XDG compliant systems. This path is irrelevant for Servers.

Reply:
I can remove all fs stuff and implement the checks, though.

I have serious doubts that anything works in case we would start to use fs for anything not on the local filesystem.